pkg/detectors/aws/aws.go

```diff
@@ -72,6 +97,12 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 		resIDMatch := strings.TrimSpace(idMatch[1])
 
+		if s.skipIDs != nil {
```

Comment:
**Nit (optional):** I think `s.skipIDs` is always non-nil since it's initialized in `New()`

Reply:
Yeah, it's probably not needed now since I made the struct not exported.